### PR TITLE
Convert selectors to MWC

### PR DESCRIPTION
**Pull request by bramkragten**
## Proposed change

Converts selectors from `paper` to `mwc` element.

Next step:
Convert the underlying elements.

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [X] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/components/ha-selector/ha-selector-select.ts [hunk 1]
```diff
@@ -18,23 +20,21 @@ export class HaSelectSelector extends LitElement {
...
-    return html`<ha-paper-dropdown-menu
-      .disabled=${this.disabled}
+    return html`<mwc-select
+      fixedMenuPosition
+      naturalMenuWidth
       .label=${this.label}
+      .value=${this.value}
+      .disabled=${this.disabled}
+      @closed=${stopPropagation}
+      @selected=${this._valueChanged}
```
Comment (balloob) [inline src/components/ha-selector/ha-selector-select.ts:30]: This doesn't seem to work

https://61fdc4fbb19bda3949bd6fca--home-assistant-gallery.netlify.app/#automation/selectors

It works currently

https://design.home-assistant.io/#automation/selectors

File: src/components/ha-selector/ha-selector-number.ts [hunk 2]
```diff
@@ -36,27 +36,23 @@ export class HaNumberSelector extends LitElement {
           >
           </ha-slider>`
         : ""}
-      <paper-input
+      <mwc-textfield
```
Comment (balloob) [inline src/components/ha-selector/ha-selector-number.ts:39]: Not new in this PR, but can we do something about the width of slider vs input?

image

**Review by balloob — DISMISSED** [review]

File: src/components/ha-selector/ha-selector-number.ts (hunk 2 above)
Comment (bramkragten) [reply src/components/ha-selector/ha-selector-number.ts:39]: Not seeing that? 
image

Firefox only?

Comment (bramkragten) [reply src/components/ha-selector/ha-selector-number.ts:39]: Yes... Firefox only

**Review by balloob — APPROVED** [review]

**Pull request merged**